packages/test/v0.7/permissionValidator.test.ts

```diff
@@ -61,235 +122,197 @@ describe("Permission kernel Account", () => {
-        })
-        // console.log("data1", data1)
-        // console.log("data3", data3)
-        // console.log("data4", data4)
-        // console.log("data5", data5)
-        console.log("data6", data6)
-        console.log("Account address:", account.address)
-        expect(account.address).toBeString()
-        expect(account.address).toHaveLength(ETHEREUM_ADDRESS_LENGTH)
-        expect(account.address).toMatch(ETHEREUM_ADDRESS_REGEX)
-        expect(account.address).not.toEqual(zeroAddress)
-    })
+            )
+
+            expect(response).toBeString()
+            expect(response).toHaveLength(TX_HASH_LENGTH)
+            expect(response).toMatch(TX_HASH_REGEX)
+            console.log("Transaction hash:", response)
+        },
+        TEST_TIMEOUT
+    )
 
     test(
-        "Smart account client send transaction",
+        "Smart account client send transaction with RateLimitPolicy ",
         async () => {
-            // const response = await kernelClient.sendUserOperation({
-            //     userOperation: {
-            //         callData: await kernelClient.account.encodeCallData({
-            //             to: zeroAddress,
-            //             value: 0n,
-            //             data: "0x"
-            //         }),
-            //         preVerificationGas: 84700n,
-            //         callGasLimit: 1273781n,
-            //         verificationGasLimit: 726789n
-            //     }
-            // })
-            const response = await kernelClient.sendTransaction({
-                to: zeroAddress,
-                value: 0n,
-                data: "0x"
+            const rateLimitPolicy = await toRateLimitPolicy({
```

Comment:
I suggest adding a more complex use case that leverages all the flags on this policy.